File: tests/go/cases/case_240_container_handoff_global_04.go
package escape_tests

var retainedCase240 = []map[string]string{}

func Case240ContainerHandoffGlobal04(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "container_handoff_global_04",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "container_handoff_global_04:" + raw,
	}
	box := map[string]map[string]string{"v": payload}
	// ESCAPE: nested container handoff retained globally.
	retainedCase240 = append(retainedCase240, box["v"])
	return "ok"
}

File: tests/rust/cases/case_240_container_handoff_global_04.rs
#![allow(unused)]

use std::collections::HashMap;
use std::sync::{Mutex, OnceLock};

#[allow(dead_code)]
static RETAINED_AUDIT: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
#[allow(dead_code)]
static RETAINED_HANDLERS: OnceLock<Mutex<Vec<String>>> = OnceLock::new();

pub fn case_240_container_handoff_global_04(input: String) -> String {
    let task_name = "container_handoff_global_04".to_string();
    let raw = if input.is_empty() { "sample".to_string() } else { input };
    let mut payload: HashMap<String, String> = HashMap::new();
    payload.insert("task".to_string(), task_name.clone());
    payload.insert("entity".to_string(), "extreme".to_string());
    payload.insert("stage".to_string(), "stress".to_string());
    payload.insert("input".to_string(), raw.clone());
    payload.insert("checksum".to_string(), format!("{}:{}", task_name, raw.len()));
    let mut outer: HashMap<String, HashMap<String, String>> = HashMap::new();
    outer.insert("v".to_string(), payload);
    // ESCAPE: nested container handoff retained globally.
    let v = outer.remove("v").expect("v");
    RETAINED_AUDIT.get_or_init(|| Mutex::new(Vec::new())).lock().expect("audit lock").push(v);
    "ok".to_string()
}
